test/e2e/tests/metrics/swaps.spec.js

```diff
@@ -94,8 +92,6 @@ async function mockSegmentAndMetaswapRequests(mockServer) {
 // TODO: (MM-PENDING) These tests are planned for deprecation as part of swaps testing revamp
 describe('Swap Eth for another Token', function () {
   it('Completes a Swap between ETH and DAI after changing initial rate', async function () {
-    const { initialBalanceInHex } = genRandInitBal();
```

Comment:
having a random balance is not needed // nor asserted in this spec, so we default to the regular seeded accounts with 25ETH